DOCS-2454: Add documentation for /charges and /payouts API requests (#1432)
MultiSafepay/docs: content/api/accounts.md

---
weight: 370
meta_title: "API - Accounts - MultiSafepay Docs"
meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
---
{{< code-block hide >}}
{{< /code-block >}}

{{< description >}}
# Accounts
<hr class="separator">

> These endpoints are used by partners and primary account holders to perform actions on affiliated accounts.

Reviewer: ```suggestion
Partners and primary account holders use these endpoints to perform actions on affiliated accounts.
```
